lib/matplotlib/table.pyi

```diff
@@ -8,12 +8,14 @@ from .transforms import Bbox
 from .typing import ColorType
 
 from collections.abc import Sequence
-from typing import Any, Literal, TYPE_CHECKING
+from typing import Any, Literal
 
 from pandas import DataFrame
 
+
```

Comment:
Please do not add blank lines that are unrelated to you actual code changes.

Reply:
it is added to remove the flake issue

Reply:
I think we do not usually apply `flake8` to `.pyi` files but am happy if anyone can confirm or correct me on that.